tests/test_css.py

```diff
@@ -14,6 +15,11 @@
         ['color'],
         'color: red;'
     ),
+    (
+        u'font-family: メイリオ; color: red; float: left; background-color: red;',
```

Comment:
So far, we've been keeping these files as ASCII and doing unicode code points for non-ASCII characters. I think we should do that here, too. We can either do that in this PR or I can do it afterwards--either is fine with me.

Reply:
I've left this.